### ISSUE TITLE
Feature request: Ignore tags inside files

### ISSUE THREAD

**Issue by dianedef** [vrtmrz/obsidian-tagfolder#101]
I love TagFolder, thank you for making and sharing it

Could we please have an option to hide all tags inside files, and only show tags that are inside the properties?

Thank you very much!
Have a great day,
Diane

**Comment by PhillipZeratul** [comment]
Yes, this would be a great option to have! It's a must have for people who want to use tags for a second and more flexible way to organize their notes. Please add this.
